src/fonduer/features/feature_libs/tabular_features.py

```diff
@@ -133,53 +130,52 @@ def _tablelib_unary_features(span: SpanMention) -> Iterator[Tuple[str, int]]:
         #      yield "COL_INFERRED_%s_[%s]" % (attrib.upper(), ngram), DEF_VALUE
 
 
-def _tablelib_binary_features(
-    span1: SpanMention, span2: SpanMention
+def _tablelib_multary_features(
+    spans: Tuple[SpanMention, ...]
 ) -> Iterator[Tuple[str, int]]:
-    """Table-/structure-related features for a pair of spans."""
+    """Table-/structure-related features for multiple spans."""
     binary_features = settings["featurization"]["tabular"]["binary_features"]
```

Comment:
Can you update this as well?

Reply:
Updates, could you re-check this just in case I missed something in the configs

Reply:
Great! I will review again once the # of features are the same in the e2e test.